plugins/KnFileUpload/dialogs/KnFileUpload.js

```diff
@@ -53,7 +53,7 @@
                     xhttp.onreadystatechange = function() {
                         if (xhttp.readyState === 4 && xhttp.status === 200) {
                             var result = JSON.parse(xhttp.responseText);
-                            uploadFile(result.result.url);
+                            uploadFile(result.url);
```

Comment:
Looks like this is broken since JSAPI removed the result wrapper from response .